src/Symfony/Component/Polyfill/Resources/stubs/Throwable.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+interface Throwable
```

Comment:
Defining this interface while Exception does not implement it does not seem like a good idea to me